example/Example.elm

```diff
@@ -9,10 +10,15 @@ import Rollbar exposing (Rollbar)
 import Task
 
 
+token : String
+token =
+    "REPLACE ME WITH YOUR TOKEN"
```

Comment:
I'm cool with taking out the example token so it doesn't work until you change it, but maybe we could add a comment like this?

```elm
-- You can use "12c99de67a444c229fca100e0967486f" as an example token to try this out
```

Reply:
TIL Rollbar has a demo account and demo tokens, how about this:
```elm
token : String
token =
    -- This is a demo token provide by Rollbar for its demo user
    -- You view and verify the result by visiting https://rollbar.com/demo/demo/
    -- This will log you in as the demo user, if you are not already logged in.
    "3cda8fbafbfe4a6599d1954b1f1a246e"

```

Reply:
Looks good! 👍